File: problems/problems_63/solution.ts
function uniquePathsWithObstacles(obstacleGrid: number[][]): number {
    
};

export function Solve(inputJsonElement: string): any {
	const inputValues: string[] = inputJsonElement.split("\n");
	const obstacleGrid: number[][] = JSON.parse(inputValues[0]);
	return uniquePathsWithObstacles(obstacleGrid);
}

File: problems/problems_63/solution.rs
use serde_json::{json, Value};

pub struct Solution;

impl Solution {
    pub fn unique_paths_with_obstacles(obstacle_grid: Vec<Vec<i32>>) -> i32 {
        
    }
}

#[cfg(feature = "solution_63")]
pub fn solve(input_string: String) -> Value {
	let input_values: Vec<String> = input_string.split('\n').map(|x| x.to_string()).collect();
	let obstacle_grid: Vec<Vec<i32>> = serde_json::from_str(&input_values[0]).expect("Failed to parse input");
	json!(Solution::unique_paths_with_obstacles(obstacle_grid))
}
